src/devices/src/virtio/net/device.rs

```diff
@@ -50,10 +54,17 @@ enum FrontendError {
     ReadOnlyDescriptor,
 }
 
-pub(crate) fn vnet_hdr_len() -> usize {
+pub(crate) const fn vnet_hdr_len() -> usize {
     mem::size_of::<virtio_net_hdr_v1>()
 }
 
+// This returns the maximum frame header length. This includes the VNET header plus
+// the maximum L2 frame header bytes which includes the ethernet frame header plus
+// the header IPv4 ARP header which is 28 bytes long.
```

Comment:
This implies that the frame hdr length returned here would only be accurate for IPv4 frames, although I don't believe that was your intention. Would the use of IPv6 be prevented in some way?

Reply:
I don't think we use IPv6 for MMDS. I'll double-check